src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothAdapterImpl.cs

```diff
@@ -479,7 +480,7 @@ internal IEnumerable<BluetoothDevice> GetBondedDevices()
                 }
                 return true;
             };
-            int ret = Interop.Bluetooth.GetBondedDevices(callback, IntPtr.Zero);
+            int ret = Interop.Bluetooth.GetBondedDevices(_bondedDeviceCallback, IntPtr.Zero);
```

Comment:
`GetBoundedDevices` is not async method, so need not change.
 anyway it also no matter

Reply:
Thank you for comment. I want to leave it as it is.